criu/sk-tcp.c

```diff
@@ -449,6 +441,18 @@ int prepare_tcp_socks(struct task_restore_args *ta)
 	return 0;
 }
 
+int restore_tcp_opts(int sk, TcpOptsEntry *toe)
```

Comment:
toe can be null here and we need to handle this case. Do I miss something?

Reply:
fixed